va/wayland/Makefile.am

```diff
@@ -53,7 +53,7 @@ protocol_source_h = \
 noinst_LTLIBRARIES		= libva_wayland.la
 libva_waylandincludedir		= ${includedir}/va
 libva_waylandinclude_HEADERS	= $(source_h)
-libva_wayland_la_SOURCES	= $(source_c) $(protocol_source_h)
```

Comment:
Please file an issue on GitHub and add Fixes #n to the commit header otherwise, I'm fine with this change.